site/app/models/OfficeHoursQueueModel.php

```diff
@@ -224,4 +254,53 @@ public function getNumberAheadInQueueToday() {
         }
         return $this->core->getQueries()->getNumberAheadInQueueToday($this->getCurrentQueueCode(), $time_in);
     }
+
+    public function getAllQueuesEver() {
+        return $this->core->getQueries()->getAllQueuesEver();
+    }
+
+    public function getQueueDataStudent() {
+        return $this->core->getQueries()->getQueueDataStudent();
+    }
+
+    public function getQueueDataOverall() {
+        return $this->core->getQueries()->getQueueDataOverall();
+    }
+
+    public function getQueueDataToday() {
+        return $this->core->getQueries()->getQueueDataToday();
+    }
+
+    public function getQueueDataByQueue() {
+        return $this->core->getQueries()->getQueueDataByQueue();
+    }
+
+    public function getQueueDataByWeekDay() {
+        return $this->core->getQueries()->getQueueDataByWeekDay();
+    }
+
+    public function getQueueDataByWeekDayThisWeek() {
+        return $this->core->getQueries()->getQueueDataByWeekDayThisWeek();
+    }
+
+    public function dayNumToDay($daynum) {
```

Comment:
Whenever possible you should add type hinting to functions to improve the php static analysis 
e.g:
```php
public function dayNumToDay(int $daynum) : string {
```